.github/workflows/actions.yml

```
@@ -0,0 +1,32 @@
+name: Continuous Integration
+on: [push]
+jobs:
+  main:
+    name: Specs
+    runs-on: [self-hosted, zendesk-stable]
+    env:
+      RAILS_ENV: test
+    strategy:
+      fail-fast: true
+      matrix:
+        ruby:
+          - 2.1
+          - 2.2
+          - 2.3
+          - 2.6
+          - 2.7 # Default
```

You might as well quote them now, and not when `3.0` installs Ruby 3.2 due to float→integer coercion.
```suggestion
          - "2.1"
          - "2.2"
          - "2.3"
          - "2.6"
          - "2.7" # Default
```